### PR TITLE
debug statements in the API path

### PR DESCRIPTION
**Pull request by maxtaco**
- we have a sesscheck deadlock somewhere, maybe this can help us narrow it down...

### CODE REVIEW

File: go/libkb/api.go [hunk 1]
```diff
@@ -189,7 +189,13 @@ func doRequestShared(api Requester, arg APIArg, req *http.Request, wantJSONRes b
 		return
 	}
 
+	dbg := func(s string) {
```
Comment (patrickxb) [inline go/libkb/api.go]: on the off chance that GetTorMode() is the culprit, can we log entry into this function?

Comment (patrickxb) [inline go/libkb/api.go]: and by this function I mean "doRequestShared"

Comment (maxtaco) [inline go/libkb/api.go]: I thought about that, but I just don't see how....

Comment (patrickxb) [inline go/libkb/api.go]: Ok, I guess we'll see the timestamps between API GET request and these new ones.

**Comment by patrickxb** [comment]
LGTM

**Comment by maxtaco** [comment]
there's no way this broke C/I, so I'm going to go ahead with it.

**Pull request merged**